kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -28,20 +28,39 @@ export const ThumbApps = [
   'healthy-living',
 ];
 
+export function getAppNameByID(id) {
+  return CustomChannelApps[id] || 'default';
+}
+
+function normalizeChannelName(channel) {
```

Comment:
Let's reuse the code we already have this time, because the logic here is slightly different than in the template. So I would move the template `getSlug()` to the components `utils`, and import it here.